controlalgorithm/angle_step_mapper.py

```diff
@@ -0,0 +1,70 @@
+"""
+Date: March 6, 2020
+Author: Sam Wu
+Contents: Map blind slat tilt angle to stepper motor steps and vice-versa
+"""
+
+from easydriver.easydriver import StepDirection
+import max_sunlight_algorithm as max_sun
```

Comment:
The build ci failures are caused by module import errors in this file. It's because python thinks that `max_sunlight_algorithm` is it's own python package but it's not. It's actually a module of the `controlalgorithm` package. To fix the import errors, you need to change the imports to absolute imports (ex. `controlalgorithm.max_sunlight_algorithm as max_sun`)

See [this article](https://realpython.com/absolute-vs-relative-python-imports/) for more explanation. Although you can also use relative imports, I think absolute imports are clearer and more in line with the rest of the codebase.

Reply:
relative/absolute imports fixed.
the smbus2 '/dev/i2c-1' errors should be fixed with your implementation of tempsensor.py